lib/watir/elements/element.rb

```diff
@@ -614,12 +657,14 @@ def assert_is_element(obj)
       end
     end
 
-    def element_call
+    def element_call(exist_check = :wait_for_exists)
+      Wait.timer ||= Wait::Timer.new.tap { |t| t.end_time = ::Time.now + Watir.default_timeout }
```

Comment:
The `||=` didn't make sense because the `#timer` method always returns a value. I committed a fix.